docs/sphinx/manual/hpx_runtime_and_resources.rst

```diff
@@ -184,13 +185,33 @@ thread pool.
 Using the resource partitioner
 ------------------------------
 
+Before |hpx| 1.5.0
+~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
..................
```

See https://hpx-docs.stellar-group.org/latest/html/contributing/documentation.html#style-guide. It's arbitrary and we're most likely not consistent everywhere, but I've tried to keep things consistent as much as possible.

Reply:
More importantly, I'd like to see the section on how one uses the resource partitioner first and have that just be the "normal" way, i.e. not mention 1.5.0 in the section title at all. That can then have a subsection on moving from older versions to 1.5.0 and newer.